util/src/main/java/com/google/cloud/hadoop/util/RetryHttpInitializer.java

```diff
@@ -38,6 +38,7 @@
 
 public class RetryHttpInitializer implements HttpRequestInitializer {
 
+  private HttpRequestInitializer delegate;
```

Comment:
Make this field final and move below static fields

Reply:
Updated